src/inc/regdisp.h

```diff
@@ -53,6 +53,11 @@ struct REGDISPLAY_BASE {
 #endif // !_TARGET_X86_ && !_TARGET_ARM
 };
 
+inline PCODE GetControlPC(REGDISPLAY_BASE *pRD) {
```

Comment:
Anyway, I'll take a look whether it is possible to unify GetRegdisplaySP and SetRegdisplaySP.